packages/manager/apps/web/client/app/hosting/statistics/hosting-statistics.controller.js

```diff
@@ -214,10 +214,18 @@ angular
           );
         }
 
+        $scope.migration = false;
+
         $q.all(getStatsPromises).then((results) => {
           $scope.model.datas = results;
           refreshChart();
+        }).catch((error)=> {
+            if(error.status >= 500){
+              $scope.migration = true;
+              return;
+            }
```

Comment:
Pay attention, you catch all errors with `status < 500` without doing anything
```suggestion
        }).catch((error)=> {
            if(error.status >= 500){
              $scope.migration = true;
              return;
            }
            throw error;
```

Reply:
done